apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/(chainPage)/components/client/FaucetButton.tsx

```diff
@@ -192,7 +192,7 @@ export function FaucetButton({
     <div className="flex w-full flex-col text-center">
       <Form {...form}>
         <form onSubmit={form.handleSubmit(claimFunds)}>
-          <Button variant="primary" className="w-full gap-2" type="submit">
+          <Button variant="primary" className="mb-2 w-full gap-2" type="submit">
```

Comment:
Can't add a bottom padding here - this will add extra spacing even if the widget is not rendered.  

either add `mt-2` on the widget or add a flex gap in between them

Reply:
lets also increase the spacing to 4 or 6 instead of 2